feedback-front/static/js/histogram.js

```diff
@@ -0,0 +1,21 @@
+function myHistogram(){
```

Comment:
give descriptive names. call this render_bar_chart(data) - it should accept the data as an argument. Make corresponding changes in the other files.